src/test/java/com/google/api/codegen/discogapic/testdata/simplecompute_config.baseline

```diff
@@ -8,17 +8,17 @@ config_schema_version: 1.0.0
 # The settings of generated code in a specific language.
 language_settings:
   java:
-    package_name: com.google.simplecompute.v1
+    package_name: com.google.cloud.simplecompute.v1
   python:
-    package_name: com.google.simplecompute_v1.gapic
+    package_name: google.cloud.simplecompute_v1.gapic
   go:
-    package_name: google.golang.org/com/google/simplecompute/v1
+    package_name: cloud.google.com/go/simplecompute/apiv1
```

Comment:
For my own understanding, could you clarify which substrings in these changed lines correspond to `org`, `model.name()` and `model.version()`?

I ask because some of these (like for Go) seem like pretty drastic changes. Are these all the correct, intended outputs?

Reply:
How can `org` contain a period, given your changes to `getPackageName`?

I assume there's  language-specific tooling to add the missing substrings, like `google.com` and `api` in `apiv1`. Is that correct, and am I correct in concluding this tooling assumes google-specifc libraries? (If so, eventually we'll want to change this to allow third-party APIs to generate gapics)

Reply:
oh you're right, I guess `org` would just be `"google"`.

The `"cloud"` string gets automatically inserted into language-specific package namers because this is GCP-centric. Practically speaking, I don't envision any non-Google API producers to use Discovery Doc, so supporting non-Google here is not a pressing concern.

Reply:
yeah, you're right we don't need to worry about non-Google Discovery. We will have to worry about non-Google gRPC, but this PR is Discovery-specific.